R/missingness.R

```diff
@@ -134,3 +135,41 @@ countMissingData <- function(x, userNAs = NULL) {
   .Deprecated("missingness", "healthcareai")
   missingness(x)
 }
+
+
+#' Replaces missing data with NA
+#'
+#' This function replaces any given value in a dataframe with NA.
```

Comment:
Can you add more detail here? Like, "This function replaces any given value in a dataframe or tibble with NA. It can be used when missing values are filled with characters like "missing".